### DISCUSSION TITLE
Providing generic type in KoGenericTypeProvider

### DISCUSSION THREAD

**Discussion by andriishpek-sigma** [LemonAppDev/konsist#1029] [Request a Feature]
### Tell us more.

`KoGenericTypeProvider` is missing a functionality to provide actual generic type (e.g. using `KoTypeDeclaration`)

**Comment by nataliapeterwas** [comment]
Thx for the feedback.

We added the ticket to the backlog: https://lemonappdev.atlassian.net/browse/KON-619

**Reply by dhs3000** [reply]
@nataliapeterwas I am also interested in this feature. In the ticket it looks like it is solved in the current / new release 0.16.0 (?)

If so, how would I use it?

Thank you :)

**Reply by nataliapeterwas** [reply]
Unfortunately, this is not fixed yet. This is a very complicated feature and we are wondering how to solve it... We forgot to change the labels in Jira before the release, sorry for this mistake...

**Reply by dhs3000** [reply]
No worries at all. I can imagine it is complicated :)

**Comment by nataliapeterwas** [comment]
Konsist `0.17.0` has been released! This release includes this improvement :) See full release log: [Release log](https://github.com/LemonAppDev/konsist/releases/tag/v0.17.0)
